db/schema.rb

```diff
@@ -126,6 +126,7 @@
     t.string "third_nationality"
     t.string "fourth_nationality"
     t.string "fifth_nationality"
+    t.integer "cycle", default: 2020
```

Comment:
Just getting into the course cycle stuff, but I thought courses are going to be year-specific. Do we need to change the CourseChoice that the candidate has applied to, rather than adding the cycle here? cc @duncanjbrown

Reply:
In Manage we determine an ApplicationChoice's cycle by looking at the course it's after/has received an offer for. So I think it wouldn't make sense to add a `cycle` to the application itself as the choices may have different cycles (eg one could be deferred) depending on their associated courses.

I think there might be a cross-team business rules conversation to have about what happens to applications near and at the cutoff dates. James has done some good analysis https://docs.google.com/document/d/1t4ekhwP3fItMpj_-5kgEQOt20d0UfLL9UdQZ94Gjrs0/edit#heading=h.c2nkonm35vkb. Going to put in a meeting for tomorrow!